### PR TITLE
Add mobilenet model

### PR DESCRIPTION
**Pull request by 1Revenger1**

### CODE REVIEW

File: camera/src/test.py
Comment (matthew-latanafrancia) [inline camera/src/test.py:64]: Does this exit out of the whole program?  If there's an error here, should we be exiting out of the software?

File: camera/src/test.py
Comment (matthew-latanafrancia) [inline camera/src/test.py:126]: Is this where the object tracking is happening? May be a nit for me, but could you add some comments just saying where that stuff is happening because I think most of us aren't familiar with OpenCV and it would help me since I'm not keeping along with all of the development haha

File: camera/src/test.py
Comment (matthew-latanafrancia) [inline camera/src/test.py:179]: When are we breaking out of the while loop? (in other words, help i don't understand what this if statement is saying)

**Review by matthew-latanafrancia — COMMENTED** [review]
I don't have a lot of experience with Python so I would like another pair of eyes on this one.  I left some questions probably due to my lack of Python experience and OpenCV experience combined.

File: camera/model/tflite_label_map.txt
Comment (1Revenger1) [reply camera/model/tflite_label_map.txt:12]: Yes, the line number/array index maps to the output from the neurel net (it just gives a float output which is rounded to an int)

File: camera/src/test.py
Comment (1Revenger1) [reply camera/src/test.py]: Likely be running it as a systemd service or smth to automatically restart. Way easier to do that then to handle restarting the camera over and over and over when it may not even be present.

**Review by matthew-latanafrancia — APPROVED** [review]
lgtm! : discussed some questions offline with avery, awaiting carters approval as well

**Review by Carter-Nakayama — APPROVED** [review]
Looks really clean. Lot of moving parts but I think everything makes sense. The tests where you use jpgs instead of videos for checking for detection makes a lot more sense than what I had in mind considering it detects per frame. 
The addition of the minimum score and 'person' label for detection is great. Looks like the class 'ImshowOutput' shows the frame output after detection so that we can see it which is awesome. Addition of the 'resolution_check.png' is also a nice touch for the JPG Source test.

**Pull request merged**
